src/pages/workspaces/workspace/tools/WorkflowView.js

```diff
@@ -0,0 +1,189 @@
+import { Agora, Dockstore, Rawls } from 'src/libs/ajax'
+import * as Nav from 'src/libs/nav'
+import * as Style from 'src/libs/style'
+import { Component } from 'src/libs/wrapped-components'
+import { tabBar } from 'src/pages/workspaces/workspace/WorkspaceTabs'
+
+
+const tableColumns = [
+  { label: 'Task name', width: 350 },
+  { label: 'Variable', width: 360 },
+  { label: 'Type', width: 160 },
+  { label: 'Attribute' }
+]
+
+const tabs = ['Inputs', 'Outputs', 'WDL']
+
+
+class WorkflowView extends Component {
+  constructor(props) {
+    super(props)
+
+    this.state = { selectedTab: 'Inputs' }
+  }
+
+  render() {
+    const { config } = this.state
+    const { workspaceNamespace, workspaceName, workflowName } = this.props
+    const workspaceId = { namespace: workspaceNamespace, name: workspaceName }
+
+    return div({ style: { display: 'flex', flexDirection: 'column', height: '100%' } }, [
+      h(TopBar, { title: 'Projects' }, [
+        div({ style: { display: 'flex', flexDirection: 'column', paddingLeft: '4rem' } },
+          [
+            div([
+              Breadcrumbs.commonElements.workspaces(),
+              Breadcrumbs.commonElements.workspace(workspaceId),
+              Breadcrumbs.commonElements.workspace(workspaceId, 'tools')
+            ]),
+            div({ style: { fontSize: '1.25rem' } }, workflowName)
+          ])
+      ]),
+      tabBar(_.merge(workspaceId, { activeTab: 'tools' })),
+      config ?
+        h(Fragment, [
+          div({
+            style: {
+              backgroundColor: Style.colors.section, padding: '1.5rem 3rem 0',
+              borderBottom: `2px solid ${Style.colors.secondary}`
+            }
+          }, [
+            this.renderSummary(),
+            this.renderTabs()
+          ]),
+          this.renderDetail()
+        ]) : [spinner({ style: { marginTop: '2rem' } })]
+    ])
+  }
+
+  async componentDidMount() {
+    const { workspaceNamespace, workspaceName, workflowNamespace, workflowName } = this.props
+
+    const config = await Rawls.workspace(workspaceNamespace, workspaceName).methodConfigs.get(workflowNamespace, workflowName)
+    this.setState({ config })
+
+    Rawls.methodConfigInputsOutputs(config).then(inputsOutputs => this.setState({ inputsOutputs }))
+  }
+
+  renderSummary = () => {
+    const { name, methodConfigVersion, rootEntityType, methodRepoMethod: { methodPath } } = this.state.config
+
+    return div({ style: { display: 'flex' } }, [
+      div({ style: { flex: '1 1 auto', lineHeight: '1.5rem' } }, [
+        div({ style: { color: Style.colors.title, fontSize: 24 } }, name),
+        div(`V. ${methodConfigVersion}`),
+        methodPath && div(`Path: ${methodPath}`),
+        div({ style: { textTransform: 'capitalize' } }, `Data Type: ${rootEntityType}`)
+      ]),
+      div({ style: { flex: '0 0 auto' } }, [
+        buttonPrimary({ disabled: true }, 'Launch analysis')
+      ])
+    ])
+  }
+
+  renderTabs = () => {
```

Comment:
TOL: it could be nice to break out the tab layout code. Even if we don't end up reusing it, it would help separate out the fiddly graphical details from the business logic.